Makefile

```diff
@@ -47,6 +48,10 @@ kv:
 ycsb:
 	$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o ycsb/ycsb ./ycsb
 
+.PHONY: tpcc
+tpcc:
+	$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o tpch/tpcc ./tpcc
```

Comment:
`tpch/tpcc` looks wrong. `tpcc/tpcc`?

Also, it's probably time to merge these rules:

```make
.PHONY: kv ycsb tpcc tpch
kv ycsb tpcc tpch:
$(GO) build -tags '$(TAGS)' $(GOFLAGS) -ldflags '$(LDFLAGS)' -v -i -o $@/$@ $@
```